src/TTreeWidget.h

```diff
@@ -26,10 +26,13 @@
 #include "pre_guard.h"
 #include <QPointer>
 #include <QTreeWidget>
+#include <QUndoStack>
+#include "dlgTriggerPatternEdit.h"
+#include "dlgTriggersMainArea.h"
 #include "post_guard.h"
```

Comment:
We don't need to put our headers in the MSVC debugging `pre/post_guard.h` "wrapper"
```suggestion
#include "dlgTriggerPatternEdit.h"
#include "dlgTriggersMainArea.h"
#include "pre_guard.h"
#include <QPointer>
#include <QTreeWidget>
#include <QUndoStack>
#include "post_guard.h"
```